framework/src/modules/token/commands/cc_transfer.ts

```diff
@@ -11,16 +11,33 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
+import { codec } from '@liskhq/lisk-codec';
+import * as cryptography from '@liskhq/lisk-cryptography';
+import { dataStructures } from '@liskhq/lisk-utils';
 import { validator } from '@liskhq/lisk-validator';
 import { BaseCommand } from '../../base_command';
 import {
 	CommandExecuteContext,
 	CommandVerifyContext,
 	VerificationResult,
 	VerifyStatus,
+	MethodContext,
 } from '../../../state_machine';
 import { TokenMethod } from '../method';
 import { crossChainTransferParams } from '../schemas';
+import { InteroperabilityMethod, TokenID } from '../types';
+import {
+	MAINCHAIN_ID,
+	MAX_DATA_LENGTH,
+	CCM_STATUS_OK,
+	CROSS_CHAIN_COMMAND_NAME_TRANSFER,
+	MODULE_NAME_TOKEN,
```

Comment:
It is better to pass the module name from module through init or constructor because name is dynamically generated by class name